### ISSUE TITLE
Serve moodist docker as a subfolder through Nginx proxy

### ISSUE THREAD

**Issue by NachoParra** [remvze/moodist#47]
Hi,

I found this project and I love it, many thanks.

I have deployed it with the docker image and I can access the webpage of the docker directly, but I want to serve it through an nginx proxy as a subfolder, smething like https://subdomain.domain.com/moodist/. 

I have tried adding a rewrite rule on the nginx configuration file
`rewrite /moodist(.*) $1 break;`
But it's not working, as expected.

Also I tried to add a environment variable on the docker container like I did with other docker images
`- "SUBFOLDER=/webtop/"`
But it's also not working.

My first question will be to ask if moodist can be served like this, and if it can be, can you please update the readme to explain how?

Thanks!!

**Comment by LarsStegman** [comment]
Have you figured out how to do it? I'm trying to do it using Traefik, but I just get a "Bad gateway" response.

**Comment by NachoParra** [comment]
No, I completely forgot this topic, as I never succeed, sorry. I will close it as I'm not investigating it anymore.

**Comment by LarsStegman** [comment]
Well, you shouldn't close the issue then, as others might have the same issue ;)
